src/features/receive/AddressBlock.tsx

```diff
@@ -30,6 +41,8 @@ export interface IAddressBlockProps {
 
   // deprecated, see Utils.getPslToUsdString
   pslPrice: number
+
+  transactions?: [ITransactionsProps]
```

Comment:
You probably meant ITransactionsProps[]?

Reply:
Yes and it has been updated